### DISCUSSION TITLE
Error on route api

### DISCUSSION THREAD

**Discussion by brandonsousa** [adonisjs/core#1324] [Help]
when I call the index of this route it loads the show from another controller that doesn't even have it, but all other methods are working normally

`Route.resource('/contact', 'ContactController').only(['index', 'store', 'show', 'destroy']).middleware('auth')`

**Comment by thetutlage** [comment]
Your question is too vague

First of all it should be `Route.resource('contact')` without the leading slash. Also, please share the complete routes file and tell which route is loaded instead.
